### DISCUSSION TITLE
Version for Vive Focus 3

### DISCUSSION THREAD

**Discussion by niiln** [korejan/ALVR#32] [General]
now focus 3 supports openxr. is it planned to release a version of the application for this headset

**Comment by korejan** [comment]
@niiln hi, some time ago I asked on the htc Dev forums if they provide their own OpenXR loader or use the cross vendor android one. I was told that the latter, if true you can install & use the generic apk (the one with no vendor name suffix), mind you this was a response from a forum admin they may have misunderstood me.

Another potential issue is, I was told sometime ago (not by HTC), when there was a private beta of the focus 3 OpenXR runtime that, at the time, it only supported gles. Currently alxr only has full support with Vulkan on Android builds.

This may have changed now, it just needs to be tested. Do you have a focus 3?

**Reply by thisleo** [reply]
i have a focus3, i installed alxr-clent.apk, but it crash

**Reply by korejan** [reply]
@thisleo Hey, yes I've been working on fixes locally, it's mostly done and working now so I will be commiting my changes in the next day or 2 with a new release.

**Reply by korejan** [reply]
@thisleo Just a heads up the code fixes have been commited, I still want to sort out a few more things before making a new offical release but if you can contact me on discord (find me on the alvr discord server).

**Comment by korejan** [comment]
I forgot to mention, even if the focus 3's OpenXR runtime does only support using gles, I do plan to either complete the GL(es) backend or do gles-vulkan interop. I just don't own the device myself to test

**Reply by niiln** [reply]
Yes, I have focus 3. I tried to install the standard version, as well as the environment loader, but to no avail. According to the latest data, OpenXR on Focus 3 is out of beta. I'm willing to help with testing.

**Reply by korejan** [reply]
@niiln Find me on the ALVR discord server (partially the same name & profile pic) send me a private message.
